packages/fluent-editor/src/modules/ai/constants.ts

```
@@ -0,0 +1,44 @@
+
+import {
```

ESLint

[error] Too many blank lines at the beginning of file. Max of 0 allowed.

(style/no-multiple-empty-lines)